dids/src/main/kotlin/web5/sdk/dids/methods/dht/DidDht.kt

```diff
@@ -202,6 +202,47 @@ public sealed class DidDhtApi(configuration: DidDhtConfiguration) {
     return BearerDid(didUri, did, keyManager, didDocument)
   }
 
+  /**
+   * Updates an existing "did:dht" DID by applying the provided changes to the DID Document.
+   *
+   * @param bearerDid The existing "did:dht" DID to update.
+   * @param options Optional parameters ([CreateDidDhtOptions]) to specify additional keys, services, and optional
+   * publishing during the update.
+   * @return The updated [BearerDid] instance.
+   */
+  public fun update(bearerDid: BearerDid, options: CreateDidDhtOptions): BearerDid {
```

Comment:
Thoughts on calling the option type passed in here `UpdateDidDhtOptions` ?

Reply:
@jiyoontbd great idea - added a public typealias for that reason (it is the same type) but with its own docs to make semantics clear. PTAL

Reply:
Cool. If we can call .Create() with CreateOptions and .Update with UpdateOptions that would be ideal

Reply:
yep, done